### ISSUE TITLE
minTime option doesn't work at all

### ISSUE THREAD

**Issue by rknetwork** [xdan/datetimepicker#385]
https://jsfiddle.net/53pkofg3/

$('.time').datetimepicker({ datepicker: false, minTime: '13:00' });

minTime has no effect at all

**Comment by rknetwork** [comment]
http://stackoverflow.com/questions/35160907/datetimepicker-mintime-has-no-effect

**Comment by EricHeesakkers** [comment]
minTime / maxTime still don't work in V2.5.1

**Comment by cybercomkvint** [comment]
Yes please fix this. I use bower to get this plugin and do not want to download it manually just to fix this bugg

**Comment by AlexArendsen** [comment]
minTime / maxTime not working for me either. Very disappointing.

**Comment by vtoulouse** [comment]
Thank you for the fix, @jmwiles3 . Do you think you could make a pull request for this?

**Comment by mav2287** [comment]
I dug into this and it is a bug. minTime and maxTime are both dead in current version. https://github.com/xdan/datetimepicker/issues/377 has information on how to do a patch.

**Comment by DaGLiMiOuX** [comment]
Any update on this? It still doesn't work in 2.5.17 and this case is like one year and a month old... Changing core files is not a solution for my case, as I get the script from cloudflare.
